pycbc/waveform/waveform.py

```diff
@@ -547,8 +548,8 @@ def get_fd_det_waveform_sequence(template=None, **kwds):
         channels, values are FrequencySeries.
     """
     input_params = props(template, **kwds)
-    input_params['delta_f'] = -1
-    input_params['f_lower'] = -1
+    if input_params['f_lower'] is None:
```

Comment:
It should be an error if f_lower is not provided, no?

Reply:
@ConWea I believe this change is actually in error. Where you are probably confused is between the get_fd_waveform and the get_fd_waveform_sequence interfaces. The sequency interface requires you to provide the frequency samples so flow / df are not supplied, whereas for the other they should be. 

I think this change is why the code examples failed.

Reply:
@ahnitz Ahh I see. Okay I will revert these changes!